pkg/oc/cli/cmd/newapp.go

```diff
@@ -385,13 +392,35 @@ func (o *NewAppOptions) RunNewApp() error {
 			fmt.Fprintf(out, "%sTrack installation of %s with '%s logs %s'.\n", indent, installing[i].Name, o.BaseName, installing[i].Name)
 		}
 	case len(result.List.Items) > 0:
+		//if we don't find a route we give a message to expose it
+		if !containsRoute {
+			//we if don't have any routes, but we have services - we suggest commands to expose those
+			svc := getServices(result.List.Items)
+			if len(svc) > 0 {
+				fmt.Fprintf(out, "%sApplication is not exposed. You can expose services to the outside world by executing one or more of the commands below:\n", indent)
```

Comment:
@bparees sorry, comment gone after squashing. If we start pooling here we will create blocking action. So should we do something like try pool for 5 s max and if it still not prepopulated not print it at all?